Lib/test/test_re.py

```diff
@@ -2338,8 +2338,8 @@ def test_possesive_repeat(self):
 ''')
 
     def test_repeat_index(self):
-        self.assertEqual(get_debug_out(r'(?:ab)*(?:cd)*'), '''\
-MAX_REPEAT 0 MAXREPEAT
+        self.assertEqual(get_debug_out(r'(?:ab)*?(?:cd)*'), '''\
+MIN_REPEAT 0 MAXREPEAT
```

Comment:
You just read my mind! I was going to propose such a change, but I thought that I was already bothering you too much.

Reply:
I just thought this after posting this PR.

> I thought that I was already bothering you too much.

As an inactive contributor, this is not a matter.
I'm not practised, so need continuous improvement the patch to get to a good state.
When I think it's good, I can always find its shortcomings afterwards.  

> I have only one question: how to prove that we need only one SRE_REPEAT structure per the REPEAT code?

I have to think about how to answer your question.